Fix typo (#16580)
mdn/content: files/en-us/web/css/filter/index.md

The `filter` property is specified as `none` or one or more of the functions listed below. If the parameter for any function is invalid, the function returns `none`. Except where noted, the functions that take a value expressed with a percent sign (as in `34%`) also accept the value expressed as decimal (as in `0.34`).

> When a single `filter` property has two or more functions its results will be different from when two or more `filter` properties are separately applied with the same functions.

Reviewer: ```suggestion
When a `filter` property has two or more functions, its results are different from the same functions applied separately using multiple `filter` properties.
```
- 'single' is implied when 'a' is used
- The original statement doesn't properly convey that we want to apply each function separately.
